### PR TITLE
[TASK] Add separate workflow for composer scripts

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #256

### CODE REVIEW

File: Documentation/Linters/GitHub/Index.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Running lints in GitHub
+=======================
+
+For GitHub, we prepared two ways of running lints. First one relies on executing
+composer scripts, and second is using exported, prepared GitHub Actions in your
+workflow.
+
+Composer scripts
+================
+You can run your lints using composer scripts. Example workflow is defined in
```
Comment (DavidBruchmann) [inline Documentation/Linters/GitHub/Index.rst:15]:
```suggestion
You can run your lints using composer scripts. An example workflow is defined in
```

File: Documentation/Linters/GitHub/Index.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+`ci-composer-scripts.yml`.
+
+Ready to use GitHub Actions
+===========================
+You can use prepared GitHub Actions. All of ready to use GitHub Actions are in
+`TYPO3 Continuous Integration organisation <https://github.com/TYPO3-Continuous-Integration>`_
+Example workflow is defined in `ci.yml`.
```
Comment (DavidBruchmann) [inline Documentation/Linters/GitHub/Index.rst:22]:
```suggestion
An example workflow is defined in `ci.yml`.
```

File: Documentation/Linters/GitLab/Index.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+.. _linters_gitlab:
+
+=======================
+Running lints in GitLab
+=======================
+In GitLab CI you should use composer scripts.
+
+Composer scripts
+================
+You can run your lints using composer scripts. Example workflow is defined in
```
Comment (DavidBruchmann) [inline Documentation/Linters/GitLab/Index.rst:12]:
```suggestion
You can run your lints using composer scripts. An example workflow is defined in
```

File: README.md [hunk 4]
```diff
@@ -139,14 +139,28 @@ ddev composer ci:ts:lint
...
 To run unit tests, type:
 
 ```bash
 composer ci:tests:unit
 ```
 
+## Running lints in CI
+### GitHub
+For GitHub, we prepared two ways of running lints. First one relies on executing
+composer scripts, and second is using exported, prepared GitHub Actions in your
```
Comment (DavidBruchmann) [inline README.md:153]:
```suggestion
1. relies on executing composer scripts
2. second is using exported, prepared GitHub Actions in your workflow.
```

File: README.md [hunk 5]
```diff
@@ -139,14 +139,28 @@ ddev composer ci:ts:lint
...
+workflow.
```
Comment (DavidBruchmann) [inline README.md:154]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -139,14 +139,28 @@ ddev composer ci:ts:lint
...
 
 To run unit tests, type:
 
 ```bash
 composer ci:tests:unit
 ```
 
+## Running lints in CI
+### GitHub
+For GitHub, we prepared two ways of running lints. First one relies on executing
```
Comment (DavidBruchmann) [inline README.md:152]:
```suggestion
For GitHub, we prepared two ways of running lints:
```

File: README.md [hunk 7]
```diff
@@ -139,14 +139,28 @@ ddev composer ci:ts:lint
...
+composer scripts, and second is using exported, prepared GitHub Actions in your
+workflow.
+#### Composer scripts
+You can run your lints using composer scripts. Example workflow is defined in
```
Comment (DavidBruchmann) [inline README.md:156]:
```suggestion
You can run your lints using composer scripts. An example workflow is defined in
```

File: README.md [hunk 8]
```diff
@@ -139,14 +139,28 @@ ddev composer ci:ts:lint
...
+`ci-composer-scripts.yml`.
+#### Ready to use GitHub Actions
+You can use prepared GitHub Actions. All of ready to use GitHub Actions are in
+[TYPO3 Continuous Integration organisation](https://github.com/TYPO3-Continuous-Integration).
+Example workflow is defined in `ci.yml`.
```
Comment (DavidBruchmann) [inline README.md:161]:
```suggestion
An example workflow is defined in `ci.yml`
```

File: README.md [hunk 9]
```diff
@@ -139,14 +139,28 @@ ddev composer ci:ts:lint
...
+### GitLab
+For GitLab, please use pipeline that is defined in `.gitlab-ci.yml`.
```
Comment (DavidBruchmann) [inline README.md:163]:
```suggestion
For GitLab, please use the pipeline that is defined in `.gitlab-ci.yml`.
```

**Review by DavidBruchmann — APPROVED** [review]
thank's for work and latest commits.

**Review by oliverklee — APPROVED** [review]

**Pull request merged**